src/Transform/BundleMemoryPools.cpp

```diff
@@ -73,7 +187,7 @@ class KrnlBundleMemoryPools : public OpRewritePattern<AllocOp> {
     if (!checkOpResultIsUsedByGetRef(&allocOp))
       return failure();
 
-    // TODO: remove once we support the bundling of dynamic memory pools.
+    // Only handle constant AllocOps.
```

Comment:
Initially I thought the dynamic case would be small enough to be part of this same rewrite rule. Not the case anymore since it is a big enough rewrite to justify a separate rewrite pattern.